ynput/OpenPype
# Max: Allows user preset the setting of preview animation in OP/AYON Setting

## Changelog Description
Allows user preset the setting of preview animation in OP/AYON Setting for review family.
- [x] Openpype
- [x] AYON
## Additional info
Ready for review after #5839 as it also touches the preview animation code base.

## Testing notes:
1. Go to OP/AYON setting
![image](https://github.com/ynput/OpenPype/assets/64118225/a324a69e-91b4-4af9-8092-0c9258f3e297)

![image](https://github.com/ynput/OpenPype/assets/64118225/c568a74b-be13-478c-a460-810d5743e825)

2.  Set your own preferences of settings
3. Launch Max via launcher
4. Create....
5. Click Review and check if the settings in the creator appears the same with the project setting
6. Publish
